addons/l10n_ar/models/account_move.py

```diff
@@ -209,11 +214,10 @@ def _inverse_l10n_latam_document_number(self):
         super()._inverse_l10n_latam_document_number()
 
         to_review = self.filtered(lambda x: (
-            x.journal_id.type == 'sale'
+            x.journal_id.l10n_ar_is_pos
             and x.l10n_latam_document_type_id
             and x.l10n_latam_document_number
             and (x.l10n_latam_manual_document_number or not x.highest_name)
-            and x.l10n_latam_document_type_id.country_id.code == 'AR'
```

Comment:
?

Reply:
I don't agree

Reply:
@jco-odoo  No problem, I will restore the line as it was
cc @jjscarafia

Reply:
@jco-odoo ready, the change has been applied and pushed

Reply:
@jco-odoo let me know if this is ready for FUN tests.
thanks!